From 8bcbef74dbac5ffca7e9083cc534d759eb65abad Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Wed, 1 Jun 2022 15:44:27 +0200
Subject: [PATCH 174/841] feat(stm32mp2): enable PMIC driver

Move file in common.mk to also build it for STM32MP2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 3487ab2f26a718c38cf509fdd5369263735592f6 feat(stm32mp2): enable PMIC driver]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: Ie6126230db22ff4072388c86696bb7c51dd11c59
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/254606
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/common/common.mk     | 1 +
 plat/st/stm32mp1/platform.mk | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/common/common.mk b/plat/st/common/common.mk
index 7c71883f2..21e128e97 100644
--- a/plat/st/common/common.mk
+++ b/plat/st/common/common.mk
@@ -142,6 +142,7 @@ PLAT_BL_COMMON_SOURCES		+=	drivers/clk/clk.c				\
 					drivers/st/clk/stm32mp_clkfunc.c		\
 					drivers/st/ddr/stm32mp_ddr.c			\
 					drivers/st/gpio/stm32_gpio.c			\
+					drivers/st/pmic/stm32mp_pmic.c				\
 					drivers/st/regulator/regulator_core.c		\
 					drivers/st/regulator/regulator_fixed.c		\
 					drivers/st/regulator/regulator_gpio.c			\
diff --git a/plat/st/stm32mp1/platform.mk b/plat/st/stm32mp1/platform.mk
index d5ab9ab06..d50e4e1e5 100644
--- a/plat/st/stm32mp1/platform.mk
+++ b/plat/st/stm32mp1/platform.mk
@@ -226,7 +226,6 @@ PLAT_BL_COMMON_SOURCES	+=	drivers/arm/tzc/tzc400.c				\
 				drivers/st/ddr/stm32mp1_ddr_helpers.c			\
 				drivers/st/i2c/stm32_i2c.c				\
 				drivers/st/iwdg/stm32_iwdg.c				\
-				drivers/st/pmic/stm32mp_pmic.c				\
 				drivers/st/pmic/stpmic1.c				\
 				drivers/st/reset/stm32mp1_reset.c			\
 				plat/st/stm32mp1/stm32mp1_context.c			\
-- 
2.47.2

